### PR TITLE
Feat/video listing

### PR DESCRIPTION
**Pull request by biswanath-tewari**
features:
1. sidebar
2. vertical video card
3. footer
4. mobile nav
5. video listing page UI

### CODE REVIEW

File: src/components/VideoCards/VerticalCard/styles.scss [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+    .VerticalCard__info {
+      .flex-wrap {
+        .VerticalCard__iconswrapper {
+          opacity: 1;
+        }
+      }
+    }
+  }
+  &__imgwrapper {
+    //width: 15rem;
```
Comment (Prabhav-Sharma) [inline src/components/VideoCards/VerticalCard/styles.scss:23]: Can remove comments, if not needed anymore

**Review by Prabhav-Sharma — APPROVED** [review]
Neat and easy to navigate code!

File: src/components/VideoCards/VerticalCard/styles.scss [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+        }
+      }
+    }
+  }
+
+  &__durationWrapper {
+    position: absolute;
+    top: 1rem;
+    right: 1rem;
+    padding: 0.15rem 0.35rem;
```
Comment (sharathnair9999) [inline src/components/VideoCards/VerticalCard/styles.scss:130]: for such a small value you could use px. just a suggestion

**Review by sharathnair9999 — COMMENTED** [review]
Looks so readable. good work

**Review by sharathnair9999 — APPROVED** [review]

**Pull request merged**